lib/egon/overcloud/undercloud_handle/deployment.rb

```diff
@@ -139,29 +128,18 @@ def edit_plan_environments(plan_name, environments)
     ## MISCELLANEOUS PLAN ACTIONS
 
     def get_plan_deployment_roles(plan_name)
-      uri = "#{base_tripleo_api_url}/plans/#{plan_name}/roles"
-      response = Fog::Core::Connection.new(uri, false).request({
-            :expects => 200,
-            :headers => {'Content-Type' => 'application/json',
-                         'Accept' => 'application/json',
-                         'X-Auth-Token' => auth_token},
-            :method  => 'GET'
-          })
-      body = Fog::JSON.decode(response.body)
-      body['roles']
-    end
-
-    def get_plan_resource_types(plan_name)
-      uri = "#{base_tripleo_api_url}/plans/#{plan_name}/resource_types"
-      response = Fog::Core::Connection.new(uri, false).request({
-            :expects => 200,
-            :headers => {'Content-Type' => 'application/json',
-                         'Accept' => 'application/json',
-                         'X-Auth-Token' => auth_token},
-            :method  => 'GET'
-          })
-      body = Fog::JSON.decode(response.body)
-      body['resource_types']
+      return ['Controller', 'Compute', 'BlockStorage', 'ObjectStorage',
+              'CephStorage']
+      #uri = "#{base_tripleo_api_url}/plans/#{plan_name}/roles"
```

Comment:
Why all this commented code? Can you refer back in git history? Or leave a comment?